vision/cdcgan_mnist/cGAN_mnist.jl

```diff
@@ -186,4 +186,6 @@ function train(; kws...)
 end    
 
 cd(@__DIR__)
-fixed_labels = train()
+if abspath(PROGRAM_FILE) == @__FILE__
+    fixed_labels = train()
```

Comment:
```suggestion
    train()
```
no need to assign the output when called as a script